.ci/vfio_jenkins_job_build.sh

```diff
@@ -0,0 +1,322 @@
+		xz -f -d "${fedora_img_cache}.xz"
+		sync
+	fi
+
+	cp -a "${fedora_img_cache}" "${fedora_img}"
+	sync
+
+	# setup cloud image
+	sudo losetup -D
+	loop=$(sudo losetup --show -Pf "${fedora_img}")
+	sudo mount "${loop}p1" /mnt
+
+	# disable selinux
+	sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /mnt/etc/selinux/config
+
+	# add intel_iommu=on to the guest kernel command line
+	sudo sed -i 's|default_kernelopts="|default_kernelopts="intel_iommu systemd.unified_cgroup_hierarchy=0|g' /mnt/boot/grub2/grub.cfg
+	sudo sed -i 's|kernelopts=|kernelopts=intel_iommu=on systemd.unified_cgroup_hierarchy=0|g' /mnt/boot/grub2/grubenv
+
+	# cleanup
+	sudo umount -R /mnt/
+	sudo losetup -d "${loop}"
+
+	qemu-img resize -f raw "${fedora_img}" +20G
+}
+
+run_vm() {
+	image="$1"
+	config_iso="$2"
+	disable_modern="off"
+	hostname="$(hostname)"
+	memory="16384M"
+	cpus=4
+
+	/usr/bin/qemu-system-${arch} -m "${memory}" -smp cpus="${cpus}" -cpu host,host-phys-bits -machine q35,accel=kvm,kernel_irqchip=split \
+	   -device intel-iommu,intremap=on,caching-mode=on,device-iotlb=on \
+	   -drive file=${image},if=virtio,aio=threads,format=raw \
+	   -drive file=${config_iso_file},if=virtio,media=cdrom \
+	   -daemonize -enable-kvm -device virtio-rng-pci -display none -vga none \
+	   -netdev user,hostfwd=tcp:${vm_ip}:${vm_port}-:22,hostname="${hostname}",id=net0 \
+	   -device virtio-net-pci,netdev=net0,disable-legacy=on,disable-modern="${disable_modern}",iommu_platform=on,ats=on \
+	   -netdev user,id=net1 \
+	   -device virtio-net-pci,netdev=net1,disable-legacy=on,disable-modern="${disable_modern}",iommu_platform=on,ats=on
+}
+
+install_dependencies() {
+	case "${ID}" in
+		ubuntu|debian)
+			# cloud image dependencies
+			deps=(xorriso curl qemu-utils openssh-client)
+
+			# QEMU dependencies
+			deps+=(libcap-dev libattr1-dev libcap-ng-dev librbd-dev gcc pkg-config libglib2.0-dev libpixman-1-dev psmisc)
+
+			sudo apt-get update
+			sudo apt-get install -y ${deps[@]}
+			;;
+		fedora)
+			# cloud image dependencies
+			deps=(xorriso curl qemu-img openssh)
+
+			# QEMU dependencies
+			deps+=(libcap-devel libattr-devel libcap-ng-devel librbd-devel gcc glib2-devel pixman-devel psmisc)
+
+			sudo dnf install -y ${deps[@]}
+			;;
+
+		"*")
+			die "Unsupported distro: ${ID}"
+			;;
+	esac
+
+	# Build and Install QEMU
+	qemu_version="4.2.0"
+	qemu_dir="${data_dir}/qemu-${qemu_version}"
+	qemu_tar_file="${data_dir}/qemu-${qemu_version}.tar.xz"
+
+	rm -rf "${qemu_dir}"
+	mkdir -p "${qemu_dir}"
+
+	pushd "${qemu_dir}"
+	[ ! -f "${qemu_tar_file}" ] && curl -Lk https://download.qemu.org/qemu-${qemu_version}.tar.xz -o "${qemu_tar_file}"
+	tar --strip-components=1 -xf "${qemu_tar_file}"
+	curl -LOk https://raw.githubusercontent.com/kata-containers/packaging/master/scripts/configure-hypervisor.sh
```

Comment:
- This is of course correct but will be easy to forget about if this is ported to 2.x. We could in fact do something similar to https://github.com/kata-containers/kata-containers/blob/2.0-dev/src/runtime/cli/release.go#L36..L44 in `lib.sh` so that both the `master` and `2.0-dev` branch versions know which repo is legacy and which is the latest.

- Why `curl -k` (aka `--insecure`)? Did you mean `curl -s`?